### PR TITLE
Fix login dialog information

### PR DESCRIPTION
**Pull request by rylandg**
Tested with CLI spec tests.

### CODE REVIEW

File: test/spec.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@
             If you don't have a key, head over to https://binaris.com to request one
             *? API Key: *
             *Authentication Succeeded*
-              (use "bn create" to create a template function in your current directory)
+              (use "bn create <runtime> <function>" to create a template function in your current directory)
```
Comment (michaeladda) [inline test/spec.yml]: IMO this is not clear. either have a working line such as `bn create node8 hello`, or `bn create node8|python2|pypy myFunction`, or instruct to look at the help of the next step, such as `bn create -help`. Using `<runtime>` is not helpful for a user who does not know which runtimes we support. Either way it has to be simple

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "binaris",
-  "version": "3.0.1",
+  "version": "3.0.3",
```
Comment (michaeladda) [inline package.json]: this is already conflicted

**Comment by rylandg** [comment]
@michaeladda I've updated this. I rebased to get the versioning correct.

**Comment by rylandg** [comment]
@michaeladda 

1. I did omit it, my bad

2. Did you read https://semver.org/? Because the fact our current version is 4.x.x and we still have many potential breaking changes down the road shows that we aren't really adhering well.

**Comment by rylandg** [comment]
@michaeladda this was updated. Any new feedback?

**Comment by rylandg** [comment]
@michaeladda ack

**Pull request merged**